public/controller/data_functions.php

```diff
@@ -352,6 +487,25 @@ function is_valid_zip_size($filename, $max_size) {
     return true;
 }
 
+function get_zip_size($filename) {
+    $size = 0;
+    $zip = zip_open($filename);
+    //
+    // FIXME: bug with this check?  Getting this warning when a non zip is uploaded
```

Comment:
So the warning happening with the FIXME comment is no longer happening? You should remove these comments then.